docs/usage.md

```diff
@@ -65,6 +65,7 @@ lat,lon,vs30
 ```
 
 ### `[calculation]`
+These two settings may be omitted, in which case the calc will be performed for all members of the corresponding Enumeration type.
```

Comment:
can we include a link to documentation listing the enum values?

Reply:
yep, done thanks.